pkgs/ffigen/lib/src/code_generator/library.dart

```diff
@@ -33,6 +33,7 @@ class Library {
     StructPackingOverride? packingOverride,
     Set<LibraryImport>? libraryImports,
     bool silenceEnumWarning = false,
+    List<String> entryPoints = const <String>[],
```

Comment:
Should this be nativeEntryPoints? or maybe document it on the field that these entry-points are only used if generating native code (and that the libraryImports are used when generating Dart code, and add a cross reference in the doc comment).

Reply:
done